tslint/tslint_annotations.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotationLevel } = require('../annotations_levels')
+
+/**
+ * Translates the tslint severity from "ERROR" | "WARNING" | "OFF"
+ * to the "HIGH", "MEDIUM" and "LOW"
+ * @param {String} severity issues severity given by tslint
```

Comment:
What is meant by "severity issues severity"

Reply:
I will paraphrase it.